src/android/com/adobe/phonegap/push/GCMIntentService.java

```diff
@@ -68,8 +68,10 @@ public void onMessageReceived(String from, Bundle extras) {
             SharedPreferences prefs = applicationContext.getSharedPreferences(PushPlugin.COM_ADOBE_PHONEGAP_PUSH, Context.MODE_PRIVATE);
             boolean forceShow = prefs.getBoolean(FORCE_SHOW, false);
             boolean clearBadge = prefs.getBoolean(CLEAR_BADGE, false);
+            String messageKey = prefs.getString(MESSAGE_KEY, "");
```

Comment:
Should default to MESSAGE

`String messageKey = prefs.getString(MESSAGE_KEY, MESSAGE);`